hack/e2e-test.sh

```diff
@@ -53,44 +53,27 @@ make deploy DEPLOY_IMAGE="${HIVE_IMAGE}"
 CLOUD_CREDS_DIR="${CLOUD_CREDS_DIR:-/tmp/cluster}"
 
 # Create a new cluster deployment
-export BASE_DOMAIN="${BASE_DOMAIN:-hive-ci.openshift.com}"
+export CLOUD="${CLOUD:-aws}"
 export CLUSTER_NAME="${CLUSTER_NAME:-hive-$(uuidgen | tr '[:upper:]' '[:lower:]')}"
-export AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID:-$(cat ${CLOUD_CREDS_DIR}/.awscred | awk '/aws_access_key_id/ { print $3; exit; }')}"
-export AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY:-$(cat ${CLOUD_CREDS_DIR}/.awscred | awk '/aws_secret_access_key/ { print $3; exit; }')}"
 export ARTIFACT_DIR="${ARTIFACT_DIR:-/tmp}"
 export SSH_PUBLIC_KEY_FILE="${SSH_PUBLIC_KEY_FILE:-${CLOUD_CREDS_DIR}/ssh-publickey}"
 export PULL_SECRET_FILE="${PULL_SECRET_FILE:-${CLOUD_CREDS_DIR}/pull-secret}"
 
 function teardown() {
-	INSTALL_JOB_NAME=$(oc get job -l "hive.openshift.io/cluster-deployment-name=${CLUSTER_NAME},hive.openshift.io/install=true" -o jsonpath='{.items[0].metadata.name}')
```

Comment:
The log capturing was moved out of the teardown so that any errors in the log capturing do not prevent the script from deleting the clusterdeployment. For some reason that I could not quickly figure out, when doing an `oc get -o jsonpath` that does not find any items, the script errors out instead of having the error contained to the if statement.